src/Schema/Schema.php

```diff
@@ -14,88 +17,263 @@
 
 class Schema
 {
+    /**
+     * @param string $getter
+     * @param string $name
+     * @param string $namespace
+     * @param bool[] $calling
+     * @param bool   $throw
```

Comment:
missing?

Reply:
`bool $throw` I forgot to remove (it used to be conditional).